server/mediaserver.go

```diff
@@ -399,7 +399,10 @@ func (s *LivepeerServer) startSession(cxn *rtmpConnection) *BroadcastSession {
 	expb.MaxInterval = BroadcastRetry
 	expb.MaxElapsedTime = 0
 	backoff.Retry(broadcastFunc, expb)
-	return sess
+	if len(sess) > 0 {
```

Comment:
Temporary, until we integrate `BroadcastSessionManager` methods into `processSegment`